compiler/packages/react-mcp-server/src/utils/runtimePerf.ts

```diff
@@ -48,10 +45,10 @@ export async function measurePerformance(code: any) {
  * @param {Object} opts - Transformation options
  * @returns {Promise<string>} - The transpiled code
  */
-async function transformAsync(ast: babel.types.Node) {
+async function transformAsync(ast: babel.types.Node, options: any) {
```

Comment:
you can fix the `any` here by just removing this function and `parseAsync`. You can inline it directly into `measurePerformance` so everything is typed correctly.